### PR TITLE
Adds 1 new test case, updates regex, does some minor formatting

### PR DESCRIPTION
**Pull request by autolyticus**
Thanks for making this plugin!

### CODE REVIEW

File: getWordCount.ts [hunk 1]
```diff
@@ -1,23 +1,28 @@
...
-    .source;
-const nonSpaceDelimitedWords = /\u3041-\u3096\u309D-\u309F\u30A1-\u30FA\u30FC-\u30FF\u4E00-\u9FD5/
-    .source;
+export default function getWordCount(text: string): number {
+  const spaceDelimitedChars =
```
Comment (kenzan100) [inline getWordCount.ts:3]: it's nothing of your responsibility, but making sense of these diff is hard for people's eyes;

what was exactly added to cover the cases of delimiters? only apostrophe?

Comment (autolyticus) [reply getWordCount.ts:3]: I didn't add anything, I treated it like machine code :D

I simply copied the latest changes from the `utils.ts` file.

Comment (kenzan100) [reply getWordCount.ts:3]: ah, you mean https://github.com/lukeleppan/better-word-count/blob/master/src/utils/StatUtils.ts okay, now I understand

**Comment by autolyticus** [comment]
Cheers!

**Pull request merged**